db-service/lib/cqn4sql.js

```diff
@@ -1563,7 +1561,7 @@ function cqn4sql(originalQuery, model) {
             as = getNextAvailableTableAlias(as)
           }
           nextStepLink.alias = as
-          whereExistsSubSelects.push(getWhereExistsSubquery(stepLink, nextStepLink, where))
+          whereExistsSubSelects.push(getWhereExistsSubquery(stepLink, nextStepLink, where, false, args))
```

Comment:
I'm not entirely sure but I can't think of a way to define a scoped query which ends in a parameterized view, the compiler does not allow associations targeting entities with parameters:
```
Error[def-unexpected-paramview-assoc]: Expected association target to have no parameters
    |
  e.cds:23:9, at entity:“Scoped”/element:“pbooks”
    |
 23 |         pbooks
    |         ^
```